pages/_app.js

```diff
@@ -13,7 +13,7 @@ const App = ({ Component }) => {
 
     React.useEffect(() => {
         if (typeof window !== 'undefined') {
-            window.pool = new Pool(process.env.WEBSOCKET_URL);
+            window.pool = new Pool(process.env.WEBSOCKET_URL, ['chat', 'drawing', 'game']);
```

Comment:
What game would be used for ?

Reply:
All the gameplay internal communication, when deciding who won, who responded first, etc

Reply:
Ok makes sense. 
Maybe I would add a channel "greetings" or "discover" which would allow name broadcast after connection. What do you think ?

Reply:
Sure, sounds good, I'll push the modification this evening

Reply:
I'll do it it's not much